config/desktop/focal/environments/i3-wm/support

```diff
@@ -1 +1 @@
-csc
+supported
```

Comment:
Is a single PR enough to change this from csc to supported?  Who is going to maintain and test this going forward?  I would think we would need someone to step up as an official maintainer for this with the responsibility to test for each release.  Is that part of the expectation here?

Reply:
@BeanGreen247 promised to take care of it.